src/components/EditorMarkdownIt.vue

```diff
@@ -143,6 +145,13 @@ export default {
 				}
 			}
 		},
+
+		setSingleCodeRule() {
```

Comment:
I am confised by the term `singleCode` (as well as the css class pendant). Does it mean
- code block (triple backticks) without a line break or
- an inline code term (single backticks)
- both?

Reply:
Naming is not my strong suit, i'll change that with the other stuff